silkworm/db/snapshots/settings.hpp

```diff
@@ -28,6 +28,7 @@ struct SnapshotSettings {
     bool enabled{true};                                                        // Flag indicating if snapshots are enabled
     bool no_downloader{false};                                                 // Flag indicating if snapshots download is disabled
     bittorrent::BitTorrentSettings bittorrent_settings;                        // The Bittorrent protocol settings
+    bool keepblocks{false};                                                    // Flag indicating if exported blocks should be kept in mdbx
```

Comment:
```suggestion
    bool keep_blocks{false};                                                    // Flag indicating if exported blocks should be kept in mdbx
```

We have to keep the flag names compatible with erigon. Unfortunately they don't follow a certain convention when it comes to word separation. Nevertheless, inside silkworm we should do it appropriately.